#### **Question:** Population has 51.1% female. In a random sample of 1,000 people, what's the probability of getting less than 48.5% or more than 53.5% female?

We use binomial distribution here, n=1000, p(probability of success) = 0.511, we want P(X < 485) + P(X > 535) = P(X <= 484) + (1 - P(X <= 535))

In [3]:
from scipy.stats import binom
n, p = 1000, 0.511
p_low = binom.cdf(484, n, p)
p_high = 1 - binom.cdf(535, n, p)
total = p_low + p_high 
print(total)

0.10736798530930072


So there is a **10.7%** probability of getting less than 48.5% or more than 53.5% female